internal/services/apimanagement/api_management_resource.go

```diff
@@ -66,547 +66,559 @@ func resourceApiManagementService() *pluginsdk.Resource {
+						Sensitive: true,
+					},
 
-						"store_name": {
-							Type:     pluginsdk.TypeString,
-							Required: true,
-							ValidateFunc: validation.StringInSlice([]string{
-								string(apimanagement.StoreNameCertificateAuthority),
-								string(apimanagement.StoreNameRoot),
-							}, false),
-						},
+					"store_name": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+						ValidateFunc: validation.StringInSlice([]string{
+							string(apimanagement.StoreNameCertificateAuthority),
+							string(apimanagement.StoreNameRoot),
+						}, false),
+					},
 
-						"expiry": {
-							Type:     pluginsdk.TypeString,
-							Computed: true,
-						},
+					"expiry": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
 
-						"subject": {
-							Type:     pluginsdk.TypeString,
-							Computed: true,
-						},
+					"subject": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
 
-						"thumbprint": {
-							Type:     pluginsdk.TypeString,
-							Computed: true,
-						},
+					"thumbprint": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
 					},
 				},
 			},
+		},
 
-			"protocols": {
-				Type:     pluginsdk.TypeList,
-				Optional: true,
-				Computed: true,
-				MaxItems: 1,
-				Elem: &pluginsdk.Resource{
-					Schema: map[string]*pluginsdk.Schema{
-						"enable_http2": {
-							Type:     pluginsdk.TypeBool,
-							Optional: true,
-							Default:  false,
-						},
+		"protocols": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Computed: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"enable_http2": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						Default:  false,
 					},
 				},
 			},
+		},
 
-			"security": {
-				Type:     pluginsdk.TypeList,
-				Optional: true,
-				Computed: true,
-				MaxItems: 1,
-				Elem: &pluginsdk.Resource{
-					Schema: map[string]*pluginsdk.Schema{
-						"enable_backend_ssl30": {
-							Type:     pluginsdk.TypeBool,
-							Optional: true,
-							Default:  false,
-						},
-						"enable_backend_tls10": {
-							Type:     pluginsdk.TypeBool,
-							Optional: true,
-							Default:  false,
-						},
-						"enable_backend_tls11": {
-							Type:     pluginsdk.TypeBool,
-							Optional: true,
-							Default:  false,
-						},
-
-						"enable_frontend_ssl30": {
-							Type:     pluginsdk.TypeBool,
-							Optional: true,
-							Default:  false,
-						},
+		"security": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Computed: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"enable_backend_ssl30": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						Default:  false,
+					},
+					"enable_backend_tls10": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						Default:  false,
+					},
+					"enable_backend_tls11": {
```

Comment:
there is an argument here to be made that we swap these final enable_* in the block to *_enabled  for consistency now